test/unit/effectsTest.js

```diff
@@ -73,32 +74,33 @@ test('other background effects', async t => {
 
 test('random background effect', async t => {
   const nativeAPI = await helpers.createDanceAPI();
-  nativeAPI.p5_.randomSeed(0);
+  sinon.stub(Math, 'random').returns(0);
 
   nativeAPI.setBackgroundEffect('rand');
-  t.equal(nativeAPI.world.bg_effect, 'kaleidoscope');
+  t.equal(nativeAPI.world.bg_effect, 'circles');
   nativeAPI.getBackgroundEffect().draw({bpm: 120});
 
   t.end();
   nativeAPI.reset();
+  sinon.restore();
 });
 
-test('rainbow foreground effect updates with specified effect', async t => {
```

Comment:
This test is testing the `rainbow` effect, which is a background, not a foreground. `floating_rainbows` is the foreground rainbow effect.